etc/init.d/lncm-online

```diff
@@ -18,17 +18,21 @@ start() {
   }
 
   ONLINE=1
+  COUNT=0
 
-  while [ $ONLINE -eq 1 ]
+  while [ $ONLINE -eq 1 ] && [ $COUNT -lt 10 ]
   do
-      echo "Checking for internet connection..."
+      echo "Checking for internet connection for 10 seconds..."
       net_check
+      COUNT=$((COUNT + 1))
       sleep 1
   done
 
   if [ $ONLINE -eq 0 ]; then
     echo "Connected to internet"
     exit 0
+  else
```

Comment:
`else` unnecessary, as `if` does `exit 0`.

And maybe better to move that check into the loop? Something like:

```sh
COUNT=0
while [ "${COUNT}" -lt "10" ]; do
	printf "Checking internet connection #%d…\t" "$(COUNT)"

	echo -e 'GET http://dl-cdn.alpinelinux.org/alpine/ HTTP/1.0\n\n' | \
		timeout 5s nc dl-cdn.alpinelinux.org 80 > /dev/null 2>&1

	if [ "$?" -ne "0" ]; then
		echo "OK!"
		exit 0
	fi
	
	echo "ERR"
	COUNT=$((COUNT + 1))
	sleep 1
done

echo "Not connected to internet after 30 seconds"
```

Reply:
sounds good but not really keen on touching stuff (and potentially breaking) thats already working. Just want to get a version of pi-factory that doesn't have any of that node stuff first